docs/examples/use_cases/paddle/resnet50/main.py

```diff
@@ -44,14 +44,18 @@ def create_dali_pipeline(batch_size, num_threads, device_id, data_dir, crop, siz
         decoder_device = 'cpu' if dali_cpu else 'mixed'
         device_memory_padding = 211025920 if decoder_device == 'mixed' else 0
         host_memory_padding = 140544512 if decoder_device == 'mixed' else 0
+        preallocate_width_hint = 5980 if decoder_device == 'mixed' else 0
+        preallocate_height_hint = 6430 if decoder_device == 'mixed' else 0
         if is_training:
             images = fn.decoders.image_random_crop(images,
-                                                   device=decoder_device, output_type=types.RGB,
-                                                   device_memory_padding=device_memory_padding,
-                                                   host_memory_padding=host_memory_padding,
-                                                   random_aspect_ratio=[0.8, 1.25],
-                                                   random_area=[0.1, 1.0],
-                                                   num_attempts=100)
+                                                device=decoder_device, output_type=types.RGB,
+                                                device_memory_padding=device_memory_padding,
+                                                host_memory_padding=host_memory_padding,
+                                                preallocate_width_hint=preallocate_width_hint,
+                                                preallocate_height_hint=preallocate_height_hint,
+                                                random_aspect_ratio=[0.8, 1.25],
+                                                random_area=[0.1, 1.0],
+                                                num_attempts=100)
```

Comment:
nitpick: indentation (line up with the first argument)

Reply:
Done